scio-elasticsearch5/src/main/scala/com/spotify/scio/elasticsearch/package.scala

```diff
@@ -64,25 +59,13 @@ package object elasticsearch {
                             maxBulkRequestSize: Int = 3000,
                             errorFn: BulkExecutionException => Unit = m => throw m)
                            (f: T => Iterable[DocWriteRequest[_]]): Future[Tap[T]] = {
-      if (self.context.isTest) {
```

Comment:
Is this code missing a `requireNotClosed` @nevillelyh ?

Reply:
Probably. Do you need to add it to the refactored code?